components/windows-service/HabService.cs

```diff
@@ -151,21 +151,11 @@ private static string LauncherPath
                 }
                 else
                 {
-                    // because we declare hab-launcher as a runtime dep
-                    // this path should exist
-                    string launcherBase = Path.Combine(Path.GetPathRoot(Environment.SystemDirectory), "hab\\pkgs\\core\\hab-launcher");
-                    string latestLauncher = LastDirectory(LastDirectory(launcherBase));
-                    return Path.Combine(latestLauncher, "bin\\hab-launch.exe");
+                    return Path.Combine(Path.GetPathRoot(Environment.SystemDirectory), @"%%LAUNCHER_PATH%%", "bin\\hab-launch.exe");
```

Comment:
I just pushed a change so that the C# always pulls the launcher path from the service config file. The `install` hook injects the correct launcher path into that config.